### PR TITLE
Support GNS TMC receivers

### PR DESCRIPTION
**Pull request by mvglasow**
This pull request adds support for the GNS FM9 receiver, different flavors of which are shipped with many navigation devices. Most of them communicate via a UART interface and can be connected to a PC via a USB-UART adapter. jSerialComm is included for serial communications.

### CODE REVIEW

**Comment by mvglasow** [comment]
Update: I’ve successfully tested this driver with the FM9BT-Y (on Android, with Qz as a frontend), so stock devices are also supported. 

One word of caution: the portable devices, which are commonly included with sat-navs, come with a pigtail or stub antenna. Getting usable data from these is a huge matter of luck. The FM9BT-Y splices into the car’s antenna and has far superior reception, almost as good as the Si470x connected to a building antenna—however, make sure the antenna amplifier in your car still gets powered (I needed to get an extra phantom power adapter for that). For the other GNS receivers, GNS offers a splitter which splices into the car antenna in the same way and has a headphone plug that plugs into the receiver instead of the supplied antenna. No experience with those, but I am expecting similar improvements.

I’ve had mixed success with a custom cable to connect the FM9 USB2USB to an antenna outlet in my home. Sometimes I get an excellent signal, then I don’t, sometimes wiggling the connector around helps, other times it doesn’t (same thing with two different cables); haven’t figured out why.

Bottom line: get the FM9BT-Y (or possibly the official antenna splitter), make sure it is installed correctly, and run your tests in the car. Other setups are unreliable.
